doc/reconstruction_models_list.rst

```diff
@@ -0,0 +1,222 @@
+.. raw:: html
+
+    <style>
+        table {
+        font-family: arial, sans-serif;
+        border-collapse: collapse;
+        width: 100%;
+        }
+
+        td, th {
+        border: 1px solid #dddddd;
+        text-align: left;
+        padding: 8px;
+        }
+
+        td.red { color: red;}
+        td.green {color: green;}
+        tr:nth-child(even) {
+        background-color: #f8f8f8;
+        }
+        </style>
+
+    <table>
+    <tr>
+        <th>Methods</th>
+        <th>Single Shell</th>
+        <th>Multi Shell</th>
+        <th>Cartesian</th>
+        <th>Ideal Data Requirements</th>
+        <th>Citations</th>
+    </tr>
+    <tr>
+        <td><a href='#diffusion-tensor-imaging'>DTI (SLS, WLS, NNLS)</a></td>
+        <td class='green'>Yes</td>
+        <td class='green'>Yes</td>
```

Comment:
I know that DTI is technically runnable on multishell data, but isn't it quite risky to say it could be used? Most multishell datasets have some shells >= 1000, and DTI begins to fail at those values.

Maybe this is better said in the data recommendation, but I still wanted to chime in.